go/ql/lib/semmle/go/frameworks/XNetHtml.qll

```diff
@@ -13,8 +13,4 @@ import go
 module XNetHtml {
   /** Gets the package name `golang.org/x/net/html`. */
   string packagePath() { result = package("golang.org/x/net", "html") }
-
-  private class EscapeString extends HtmlEscapeFunction {
-    EscapeString() { this.hasQualifiedName(packagePath(), "EscapeString") }
-  }
```

Comment:
Why delete this?

Reply:
This never did anything, but I've restored it and fixed it in https://github.com/github/codeql/pull/12562/commits/f521e661d7eb5dbc71c3d47c0a788f3e5800841e